libblockverifier/ExecutiveContext.cpp

```diff
@@ -144,6 +144,10 @@ std::shared_ptr<std::vector<std::string>> ExecutiveContext::getTxCriticals(const
         if (p->isParallelPrecompiled())
         {
             auto ret = make_shared<vector<string>>(p->getParallelTag(ref(_tx.data())));
+            for (string& critical : *ret)
```

Comment:
for what ?

`critical` isn't used anywhere

Reply:
Append precompiled contract's address in criticals, let other contract not to influence this precompiled contract

Reply:
This is a small bug fix. Has no relation with disk encryption feature.